postgres/datadog_checks/postgres/postgres.py

```diff
@@ -3,38 +3,46 @@
 # Licensed under Simplified BSD License (see LICENSE)
 import re
 import socket
+import string
 import threading
 from contextlib import closing
 
-import pg8000
+import psycopg2
```

Comment:
Hi, would you mind updating your branch with the latest master? The diff is showing changes that it should not.